<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtCopyDialog</class>
 <widget class="QWidget" name="qtCopyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
   <item>
    <widget class="QLabel" name="qtTitleLabel">
     <property name="text">
      <string>Copy DB</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="qtDatabasePathComboBox"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="qtDirectoryPushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="qtDatabasePushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QScrollArea" name="qtDatastreamsScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="qtDatastreamsListWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>569</width>
          <height>242</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="qtDatastreamsListLayout"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="qtDirectoryLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="qtNameLineEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="qtCopyAllCheckBox">
       <property name="text">
        <string>Copy all datastreams</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Database</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="qtCopyPushButton">
     <property name="text">
      <string>Make a copy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="qtProgressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
